app/src/androidTest/java/com/github/onlynotesswent/ui/overview/OverviewTest.kt

```diff
@@ -267,16 +299,6 @@ class OverviewTest {
 
   @Test
   fun dragAndDropNoteWorksCorrectly() {
-    `when`(noteRepository.getRootNotesFrom(eq("1"), any(), any())).then { invocation ->
```

Comment:
Why did you remove this test?

Reply:
I removed it by error, the latest push should reput it.